cmd/acr/purge.go

```diff
@@ -103,6 +103,10 @@ func newPurgeCmd(out io.Writer, rootParams *rootParameters) *cobra.Command {
 			if err != nil {
 				return err
 			}
+			// A clarification message for --dry-run.
+			if purgeParams.dryRun {
+				fmt.Println("ATTENTION: The following output shows what WOULD be deleted if the purge command was executed. Nothing is deleted.")
```

Comment:
How about change "ATTENTION: ..." to "DRY RUN: ..."?

Reply:
changed.

Reply:
changed.